### ISSUE TITLE
Reduce font-size of intersection name on rank card

### ISSUE THREAD

**Issue by danrademacher** [GreenInfo-Network/nyc-crash-mapper-chart-view#87]
Client request:
>1- in the ranking page, could we reduce the size of the characters for the addresses? BUT NOT the borough…  it shoudl be 20% smaller. 

Not sure I get why she wants that, but some differentiation here is nice, and sounds easy enough to give it to her since we already split the borough and intersections out.

Here's a span around intersections at 14px (about 80% of 18px):
![image](https://user-images.githubusercontent.com/1423200/35941703-0329ab84-0c08-11e8-9443-b9f78b5bcd34.png)

**Comment by gregallensworth** [comment]
I added a sub-title behavior sort of thing to it, so a second line would be at `1.4rem` instead of the `1.8rem` used in the rest of the title.

I also reduced the `line-height` of these titles, so there's not as much empty space as in the depiction above.

Examples below, of Neighborhood rankcards (no subtitle) and Intersection rankcards (with subtitle).

<img width="716" alt="2018-02-15 07_29_53-nyc crash mapper" src="https://user-images.githubusercontent.com/3117633/36264640-1745e73e-1222-11e8-98ff-ffe99fdf33dc.png">
<img width="953" alt="2018-02-15 07_30_09-nyc crash mapper" src="https://user-images.githubusercontent.com/3117633/36264641-1766f46a-1222-11e8-8391-16be0cc30059.png">
